crates/api/src/agent.rs

```diff
@@ -190,6 +165,7 @@ pub struct AgentInfo {
     pub url: Option<Url>,
 
     /// The arc over which this agent claims authority.
+    #[serde(default = "BasicArc::default")]
```

Comment:
Should this be a required field? It was naturally optional due to being an `Option` type

Reply:
See my other comment about the untagged serialization. But I think it's still correct if a peer doesn't specify an arc that we assume that peer does not claim any authority == Empty.